### PR TITLE
fix(button): submit button when in form

### PR DESCRIPTION
**Pull request by skhamvon**

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-storybook#236.0 ✘  
* 
  * build ✘

**Comment by ovh-cds** [comment]
CDS Report build-storybook#238.0 ✘  
* 
  * build ✘

**Comment by ovh-cds** [comment]
CDS Report build-storybook#239.0 ✘  
* 
  * build ✘

File: packages/contributing/contributing-deployment-and-versioning.mdx [hunk 1]
```diff
@@ -69,27 +70,44 @@ yarn ods:patch
 ### Minor: add extra features
 
 You added one or several features **without any breaking changes** in the branch `release/next`.
-You have to release it from the `release/next` branch:
+Create a release branch from up-to-date `release/next` that corresponds to the version you are going to release like `release/X.Y`.
```
Comment (stevering) [inline packages/contributing/contributing-deployment-and-versioning.mdx:73]:
```suggestion
Create a release branch from up-to-date `master` that corresponds to the version you are going to release like `release/X.Y`.
```

File: packages/contributing/contributing-deployment-and-versioning.mdx [hunk 2]
```diff
@@ -69,27 +70,44 @@ yarn ods:patch
...
-Then, create a release branch from the commit tag that correspond to the version like `release/X.Y`.
+The release branch will be used for bugfixes later (patch).
 It will be used for bug fixes later (patch).
 Once the support for the version has ended, the branch will be un-referenced.
 
 ### Major: breaking changes
 
-You added breaking changes in the `release/next` branch.
-You have to generate a major version from `release/next` branch.
+You added breaking changes in the `release/next` branch:
```
Comment (stevering) [inline packages/contributing/contributing-deployment-and-versioning.mdx:87]:
```suggestion
You added breaking changes in the `master` branch:
```

File: packages/contributing/contributing-deployment-and-versioning.mdx [hunk 3]
```diff
@@ -69,27 +70,44 @@ yarn ods:patch
...
+
+Create a release branch from up-to-date `release/next` that corresponds to the version you are going to release like `release/X.Y`.
```
Comment (stevering) [inline packages/contributing/contributing-deployment-and-versioning.mdx:89]:
```suggestion
Create a release branch from up-to-date `master` that corresponds to the version you are going to release like `release/X.Y`.
```

**Review by stevering — CHANGES_REQUESTED** [review]
this PR is started from master for a fix into release/12.0.
it is not the good workflow.
you have to start from release/12.0

**Pull request closed without merging**